input/pagecontent/RAD-Y1.md

```diff
@@ -183,7 +189,7 @@ The Sender may encode code-able content in findings or impressions using the app
 
 The Sender shall encode the study in which this observation is derived from, if available, in Observation.derivedFrom using an [IMR ImagingStudy](StructureDefinition-imr-imagingstudy.html) Resource. This attribute shall include the study and series references. IMR ImagingStudy may include image references.
 
-For narrative content, the Sender can directly embed one or more image references inline the text.
+For narrative content, the Sender may directly embed one or more image references inline the text.  TO DO:  Does this sentence belong under IMR DiagnosticReport ?? 
```

Comment:
No. It belongs to Observation because the narrative content is part of Observation.